src/main/java/uk/gov/hmcts/reform/iacasedocumentsapi/domain/entities/DocumentTag.java

```diff
@@ -63,6 +63,7 @@ public enum DocumentTag {
     INTERNAL_HO_FTPA_DECIDED_LETTER("internalHoFtpaDecidedLetter", CaseType.ASYLUM),
     INTERNAL_HEARING_REQUIREMENTS_UPDATED_LETTER("internalHearingRequirementsUpdatedLetter", CaseType.ASYLUM),
     INTERNAL_MAINTAIN_CASE_LINKS_LETTER("internalMaintainCaseLinksLetter", CaseType.ASYLUM),
+    INTERNAL_UPLOAD_ADDITIONAL_EVIDENCE_LETTER("internalUploadAdditionalEvidenceLetter", CaseType.ASYLUM),
```

Comment:
Dont think we need to prefix with internal. If a user uploads additional evidence, it will either be for internal case or not. There will never be a situation where 2 documents exist where one is for internal upload additional evidence, and the other is just uploadAdditionalEvidence (for LR/AIP journey).

Slightly pedantic, but it avoids cluttering up doc tags and separating doc tags out (distinguishing between internal or not). Ive approved regardless.

Reply:
This tag isn't for the actual additional evidence being uploaded but for the letter attached to the notification sent to DET to be printed off